.github/workflows/ci.yml

```diff
@@ -26,6 +26,22 @@ jobs:
           cache: 'pnpm'
       - run: pnpm install
       - run: pnpm turbo lint prettier
+  dev-script:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+      - uses: pnpm/action-setup@v2
+        with:
+          version: 8
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: 'pnpm'
+      - run: pnpm install
+      - run: pnpm start-server-and-test 'pnpm dev --filter=registry' 3002 'exit 0;'
+        working-directory: apps/registry
```

Comment:
The tests do not run, since I pushed all commits at once. But if this test would run, it would fail because `pnpm dev --filter=registry` only works in the root of the project, not in `apps/registry` for example. With the changes from https://github.com/jsonresume/jsonresume.org/pull/26/commits/390f8b60138a4633ebb3014a4100903a023204cb it does not matter where one is when executing `pnpm turbo dev --filter=registry`.